### PR TITLE
upgrade pandas version to 2.2.3

### PR DESCRIPTION
**Pull request by PrabhasKalyan**
# Pull Request

upgrading pandas to version 2.2.3

### CODE REVIEW

**Comment by peterdudfield** [comment]
Its failing, becaue you need to add alembic migration to this PR aswell. Do you need any help doing that?

**Comment by peterdudfield** [comment]
let me know when you. are ready for me to review, thank you for doing all this

File: alembic/versions/575f99caf2d4_solve_issues_active_flag_to_sites_166_.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+Create Date: 2024-11-25 20:43:55.866383
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '575f99caf2d4'
+down_revision = 'ea0672033541'
```
Comment (peterdudfield) [inline alembic/versions/575f99caf2d4_solve_issues_active_flag_to_sites_166_.py:14]: oh, and becasue of https://github.com/openclimatefix/pv-site-datamodel/pull/180, you might need to update this - sorry

Comment (PrabhasKalyan) [reply alembic/versions/575f99caf2d4_solve_issues_active_flag_to_sites_166_.py:14]: would you please tell me what i need to do further

Comment (peterdudfield) [reply alembic/versions/575f99caf2d4_solve_issues_active_flag_to_sites_166_.py:14]: I think if you 
1. change down_version to d5c9e13a8b66
2. merge `main` branch back in

**Comment by peterdudfield** [comment]
Thanks @PrabhasKalyan for this. Would you be able to add one or two test for the `get_api_requests_for_one_user` function with the new inputs

**Comment by peterdudfield** [comment]
This helps with #168 #166 #173

**Review by peterdudfield — APPROVED** [review]

**Review by peterdudfield — CHANGES_REQUESTED** [review]
Thanks so much for doing all this, is there a missing change in https://github.com/openclimatefix/pv-site-datamodel/blob/main/pvsite_datamodel/sqlmodels.py#L120. Should add the `active` column

**Pull request merged**